pkg/services/oracle/neofs/neofs.go

```diff
@@ -149,7 +155,7 @@ func getHash(ctx context.Context, c *client.Client, addr *object.Address, ps ...
 	if len(hashes) == 0 {
 		return nil, fmt.Errorf("%w: empty response", ErrInvalidRange)
 	}
-	return hashes[0][:], nil
+	return util.Uint256(hashes[0]).MarshalJSON()
```

Comment:
It's wrapped into object which `MarshalJSON()` won't do IIRC.

Reply:
It is string:
https://github.com/neo-project/neo-modules/blob/master/src/OracleService/Protocols/OracleNeoFSProtocol.cs#L119

Reply:
OK, I guess I can't read C# interpolated strings properly.